api/src/client/AirtableFeedbackClient.ts

```diff
@@ -0,0 +1,52 @@
+import { LogWriterType } from "@libs/logWriter";
+import { FeedbackRequest } from "@shared/feedbackRequest";
+import { UserData } from "@shared/userData";
+import Airtable from "airtable";
+import { FeedbackClient } from "../domain/types";
+
+type AirtableConfig = {
+  apiKey: string;
+  baseId: string;
+  baseUrl: string;
+};
+
+export const AirtableFeedbackClient = (config: AirtableConfig, logWriter: LogWriterType): FeedbackClient => {
+  const table = "tblHZp9vjRIdooR4Z";
```

Comment:
This should be moved to an env variable, so that it's more easily configurable

Reply:
Updating the table variable to table name and not table id. Therefore this no longer needs to be a env variable right?

Reply:
will be address as part of the chore